Context/Config.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
```

Comment:
This is what replaces withPageMetaData and withCurrentDate. The logic here should cover what they did previously.